homeassistant/components/motioneye/config_flow.py

```diff
@@ -42,13 +43,18 @@ def _get_form(
             user_input: dict[str, Any], errors: dict[str, str] | None = None
         ) -> FlowResult:
             """Show the form to the user."""
+            url_schema: dict[vol.Required, type[str]] = {}
+            if not self._hassio_discovery:
+                # Only ask for URL when not discovered
+                url_schema = {
```

Comment:
We could set the value in the existing empty dict instead of creating a new dict, right?

Reply:
Yes, we could. 🤷 

I found this a more readable solution, as it keeps the schema in the original place.

Reply:
I'm not following about keeping the schema in the original place.

It's nice to avoid creating new instances, generally.

Reply:
I misunderstood...
Adjusting.

Reply:
I hope I understood it right :)